### ISSUE TITLE
[Quick Answers][Accessibility] The results cannot be accessed while Voice-Over is on

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#34639]
### Steps to reproduce

1. Open Firefox and turn on Voice-Over.
2. Interact with Quick Answer.
3. Tap on a result.

### Expected behavior

The button is announced and the result is accessed.

### Actual behavior

There is no visual feedback upon tapping, and when tapped, the result page is not opened.

### Device & build information

- **Device:** iPhone 14 Pro Max (26.2)
- **Build version:**  v9000 (74090)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-16281)
